### ISSUE TITLE
Dual-boot issue with Windows 11 on another drive

### ISSUE THREAD

**Issue by devhead67** [linuxmint/mint22.1-beta#98]
I have Windows 11 on a separate NVMe drive, and even when I tried to install this Linux Mint beta and putting everything entirely on another NVMe drive, it would create the boot partition on the correct drive as the Mint installation, but it always copied the boot folder and files to the EFI partition on my other drive where Windows was.  The EFI partition on the install drive was empty.
After installation it boots up after the Grub screen, then the screens would just be blank forever, and I would have to restart it.

**Comment by devhead67** [comment]
FWIW, I tried installing it 3 times (always doing the manual installation process, and never touching the NVMe drive with Windows 11), but it always installed the boot files on the Windows EFI boot partition.

**Comment by Rockets31** [comment]
This is a known bug for years... This happens every time when I install linuxmint on external usb-testing drive. Before installing I always need to rename '/EFI/ubuntu' folder (of my running PC), then it can not be overwritten... After installation I copy /EFI/ubuntu to external usb-drive, edit fstab entry to usb-drive & rename original '/EFI/ubuntu' folder.
I think they won't fix it, but didn't they talk about something like a brand-new mintinstaller in those days 😲!?

**Comment by devhead67** [comment]
Wow, that is very interesting.  The odd thing is, I have NEVER had this issue with the current release of Mint.  Squirrely...

**Comment by Rockets31** [comment]
Upstream already dropped the installer (ubiquity), so on the long run they usually will do something about it... They already have their own installer for linuxmint debian edition, maybe it will be extended to ubuntu based version, too. For sure, mint team will let users inform about it.

**Comment by Hawaiihemd** [comment]
Yep, this is a bug that pops up quite often in the forum. People usually go out of their way to unplug drives to work around this.

I suggest to add a release note adressing this.
